src/index.js

```diff
@@ -1,4 +1,152 @@
-const $ = document.querySelector.bind(document)
 
+/* global document, fetch, localStorage */
+/* eslint no-undef: "error" */
+/* eslint prefer-arrow-callback: "error" */
+/* eslint-disable no-use-before-define: "error" */
+/* eslint-disable indent: "error" */
+// eslint-disable-next-line no-undef
+const $ = document.querySelector.bind(document);
 
-$('h1').style.color = 'blue';
\ No newline at end of file
+
+const convertToCelsius = (k) => {
+	const value = parseFloat((k - 273.15).toFixed(6));
+	return value;
+};
+
```

Comment:
let's add a way on the UI to toggle between celsius to fahrenheit.

Reply:
done